sourcecode/apis/contentauthor/resources/assets/react/components/QuestionContentType/components/QuestionCard/components/AddAnswer/index.js

```diff
@@ -28,17 +29,16 @@ const AddAnswer = props => {
         },
     };
 
+    const intl = useIntl();
+
     return (
-        <div className="addAnswerContainer">
-            <div style={styles.dividerContainer}>
-                <Button
-                    onClick={onClick}
-                    style={styles.button}
-                    aria-label={label}
-                >
-                    {icon}
-                </Button>
-            </div>
+        <div className="addAnswerContainer" onClick={onClick} style={styles.dividerContainer}>
```

Comment:
Move the CSS styles to the CSS class used, button styles can also be moved. The `styles.divider` seems unused.